src/shared/clients/ecsClient.ts

```diff
@@ -77,6 +77,19 @@ export class DefaultEcsClient {
         }
     }
 
+    public async listTasks(cluster: string, serviceName: string): Promise<string[]> {
+        const sdkClient = await this.createSdkClient()
+
+        const params: ECS.ListTasksRequest = { cluster: cluster, serviceName: serviceName, desiredStatus: 'RUNNING' }
+        try {
+            const listTasksResponse = await sdkClient.listTasks(params).promise()
+            return listTasksResponse.taskArns ?? []
+        } catch (error) {
+            getLogger().error(`ecs: Failed to get tasks for Cluster ${cluster} and Service ${serviceName}: `)
```

Comment:
variable values should go in quotes, if they must be in the middle of the message. This is mentioned in [guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/docs/CODE_GUIDELINES.md) (and previous PRs).
```suggestion
            getLogger().error(`ecs: Failed to get tasks for Cluster "${cluster}" and Service "${serviceName}": `)
```

alternatively you could use this feature of console.log / Winston:

            getLogger().error(`ecs: Failed to get tasks for Cluster %O and Service %O: %O`, cluster, serviceName, error)

BTW, it is not really valid to assume that the `throw error` will sink to the log, so typically we log the error explicitly (thus `error` is given in the sample above).